src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -381,6 +383,8 @@ private async Task<ActionResult<SynchronizePullRequestResult>> SynchronizePullRe
                     {
                         await UpdateSubscriptionsForMergedPRAsync(pr.ContainedSubscriptions);
                     }
+
+                    await AddDependencyFlowEventsAsync(pr.ContainedSubscriptions, "Completed", $"{status} Manually - {pr.MergePolicyResult}");
```

Comment:
Do you anticipate having many `flowEvent`s? If so it might make sense to create an enum.

Reply:
There will only be 3 flow events: Created, Updated, and Completed.

There will be many FlowReasons.

Reply:
Alright, I don't think that is too big of a deal for events but might for FlowReasons? I am just trying to avoid things that have bitten us in the past which by having "free text" stuff around, when we try to map stuff, things fail since we have, for example, "Complete" in one side and "Completed" in the other.

Reply:
Makes sense. I'll switch things around to use some enums. I think that's smart, for both.